### PR TITLE
Flw 2106 scam warnings UI

### PR DESCRIPTION
**Pull request by DRadmir**

### CODE REVIEW

File: fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewLayout.swift [hunk 1]
```diff
@@ -109,12 +122,19 @@ final class ChooseRecipientViewLayout: UIView {
...
-            make.leading.trailing.bottom.equalToSuperview()
+        let contentVStackView = UIFactory.default.createVerticalStackView()
+
+        addSubview(contentVStackView)
+        contentVStackView.snp.makeConstraints { make in
+            make.top.equalTo(searchView.snp.bottom).offset(UIConstants.verticalInset)
+            make.leading.trailing.equalToSuperview().inset(UIConstants.horizontalInset)
         }
 
+        scamWarningView.isUserInteractionEnabled = true
```
Comment (Viscount88) [inline fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewLayout.swift:133]: Remove .isUserInteractionEnabled and .isHidden to scamWarningView var init

Why you should set user interaction?

File: fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewLayout.swift [hunk 2]
```diff
@@ -77,6 +79,7 @@ final class ChooseRecipientViewLayout: UIView {
         didSet {
             if locale != oldValue {
                 applyLocalization()
+                scamWarningView.locale = locale
```
Comment (bnsports) [inline fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewLayout.swift:82]: why not inside applyLocalization?

File: fearless/Modules/NewWallet/WalletSend/WalletSendPresenter.swift [hunk 3]
```diff
@@ -56,7 +59,8 @@ final class WalletSendPresenter {
             tipRequired: chainAsset.chain.isTipRequired,
             tipViewModel: provideTipViewModel(),
             feeViewModel: provideFeeViewModel(),
-            amountInputViewModel: provideInputViewModel()
+            amountInputViewModel: provideInputViewModel(),
+            scamInfo: scamInfo
```
Comment (Viscount88) [inline fearless/Modules/NewWallet/WalletSend/WalletSendPresenter.swift:63]: You put it to viewModel just to get it from this viewModel without any changes in the "bind" method. You can just take it from own properties directly.

File: fearless/Modules/NewWallet/WalletSendConfirm/WalletSendConfirmPresenter.swift [hunk 4]
```diff
@@ -137,6 +142,30 @@ final class WalletSendConfirmPresenter {
...
+            return
+        }
+
+        let closeAction = SheetAlertPresentableAction(
+            title: R.string.localizable.commonClose(preferredLanguages: selectedLocale.rLanguages),
+            style: UIFactory.default.createMainActionButton(),
+            handler: nil
+        )
+        let title = scamInfo.type.description(for: selectedLocale, assetName: chainAsset.asset.name)
+        let sheetViewModel = SheetAlertPresentableViewModel(
```
Comment (bnsports) [inline fearless/Modules/NewWallet/WalletSendConfirm/WalletSendConfirmPresenter.swift:156]: to viewmodelfactory

**Review by bnsports — CHANGES_REQUESTED** [review]

File: fearless/Common/View/ScamWarningExpandableView.swift [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+    // MARK: - Actions
+
+    @objc private func handleTapGesture() {
+        let isOpen = indicator.isActivated
+        let offset = isOpen ? -Constants.expandViewHeight : 0
+        expandableCloudView.snp.updateConstraints { make in
+            make.top.equalTo(mainCloudView.snp.bottom).offset(offset)
+        }
+
+        _ = isOpen ? indicator.deactivate() : indicator.activate()
```
[the remark this answers is not in the document]
Comment (DRadmir) [reply fearless/Common/View/ScamWarningExpandableView.swift:229]: Then need disable SwiftLint rules

File: fearless/Modules/NewWallet/WalletSend/WalletSendPresenter.swift (hunk 3 above)
Comment (DRadmir) [reply fearless/Modules/NewWallet/WalletSend/WalletSendPresenter.swift:63]: this is your module, why you don't refactor WalletSendViewState for receive separately viewModels?

**Review by bnsports — APPROVED** [review]

**Pull request merged**